tests/sles4sap/trento/test_trento_web.pm

```diff
@@ -17,22 +17,17 @@ sub run {
     die "Only AZURE deployment supported for the moment" unless check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE');
     $self->select_serial_terminal;
 
-    my $machine_ip = $self->az_get_vm_ip;
-
-    my $trento_web_password_cmd = $self->az_vm_ssh_cmd(
-        'kubectl get secret trento-server-web-secret' .
-          " -o jsonpath='{.data.ADMIN_PASSWORD}'" .
-          '|base64 --decode', $machine_ip);
-    my $trento_web_password = script_output($trento_web_password_cmd);
+    my $machine_ip = $self->get_trento_ip;
+    my $trento_web_password = $self->get_trento_password;
 
     my $cypress_test_dir = "/root/test/test";
     enter_cmd "cd " . $cypress_test_dir;
     my $cypress_env_cmd = './cypress.env.py' .
       " -u http://$machine_ip" .
       " -p $trento_web_password" .
       ' -f Premium';
-    if (get_var("TRENTO_VERSION")) {
-        $cypress_env_cmd .= ' --trento-version ' . get_var("TRENTO_VERSION");
+    if (get_var('TRENTO_VERSION')) {
```

Comment:
I know you just changed the quotes, but `TRENTO_VERSION` does not make sense to me, the trento version does not require variable to find out what trento version is running.

Reply:
This line of code is in charge to tell to the cypress engine (used to test the web interface of Trento) which string to expect about the version on the web interface.
This variable usage can be improved here and in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15419/files#diff-9616afe1f4b58be89b042ba489a6f886f4ff68c43e9c1cdb613ccd885078a5a7L43